samples/MegaDrone/src/main/cpp/native-lib.cpp

```diff
@@ -41,23 +41,23 @@ extern "C"
 JNIEXPORT void JNICALL
 Java_com_example_oboe_megadrone_MainActivity_startEngine(JNIEnv *env, jobject instance,
                                                          jintArray jCpuIds) {
-
+    engine = new AudioEngine;
     std::vector<int> cpuIds = convertJavaArrayToVector(env, jCpuIds);
-    engine.start(cpuIds);
+    engine->start(cpuIds);
 }
 
 extern "C"
 JNIEXPORT void JNICALL
 Java_com_example_oboe_megadrone_MainActivity_stopEngine(JNIEnv *env, jobject instance) {
 
-    engine.stop();
-
+    engine->stop();
+    //delete engine;
```

Comment:
Where does this get deleted? Could we use a unique_ptr<>?

Reply:
This is what I noted in the issue, I was unsure where the engine was being deleted, but when I attempted to delete again, I hit mem access errors.